packages/docs-app/src/examples/core-examples/audio/oscillator.ts

```diff
@@ -17,10 +17,7 @@
 export class Oscillator {
     public oscillator: OscillatorNode;
 
-    public constructor(
-        private context: AudioContext,
-        freq: number,
-    ) {
+    public constructor(private context: AudioContext, freq: number) {
```

Comment:
there have been some build system changes lately, so pls make sure to pull in develop and run `yarn` at the root of the repo - you need to upgrade your version of `prettier`